uploader.json

```diff
@@ -1642,6 +1642,11 @@
             "pattern": [],
             "symbolic_name": "docker_sysconfig"
         },
+        {
+            "file": "/var/log/libvirt/libvirtd.log",
+            "pattern": [],
+            "symbolic_name": "libvirtd_log"
+        },
```

Comment:
This entry should be put in alpha sorted order based on the symbolic name.

Reply:
This is removed and will be added as a separate specs.